app/router.js

```diff
@@ -7,4 +7,8 @@ export default class Router extends EmberRouter {
 }
 
 Router.map(function() {
```

Comment:
`map(function(){})` vs 
`map(()=>{})`

What do you prefer?

Reply:
this was auto generated by ember cli, so I will leave it as function!